### PR TITLE
Favicon_navbarBugs_Hover

### PR DESCRIPTION
**Pull request by Code-eat-sleep-include**
## Description

Added a Favicon. The SignUp text stayed black in black bg when clicked, so fixed that color. Added a hover effect over the items in navbar. And added a small logo on the top left corner in navbar. 

### Changes

Added a Favicon. Fixed signup. Added a hover effect over the items in navbar. And added a small logo on the top left corner in navbar. 

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: client/src/assets/shoelogo.jpg
Comment (dvjsharma) [inline client/src/assets/shoelogo.jpg:1]: We already have a logo named `banner-hero.png`. Please use that.

File: client/src/components/Navbar/navbar.jsx [hunk 1]
```diff
@@ -162,27 +175,25 @@ const Navbar = () => {
...
                             Login
                         </button>{" "}
                     </Link>
                     <Link to="signup">
                         {" "}
-                        <button className="inline-block bg-black  rounded-full border-2 border-neutral-800 px-6 pb-[6px] pt-2 text-[12px] font-medium uppercase leading-normal text-white transition duration-150 ease-in-out hover:border-neutral-800 hover:bg-white hover:text-black focus:border-neutral-800 focus:text-neutral-800 focus:outline-none focus:ring-0 active:border-neutral-900 active:text-neutral-900 ">
+                        <button className="inline-block bg-black  rounded-full border-2 border-neutral-800 px-6 pb-[6px] pt-2 text-[12px] font-medium uppercase leading-normal text-white transition duration-100 ease-in-out hover:border-neutral-800 hover:bg-white hover:text-black focus:border-neutral-800 focus:text-neutral-800 focus:outline-none focus:ring-0 active:border-neutral-900 active:text-neutral-900 focus:text-white">
```
Comment (dvjsharma) [inline client/src/components/Navbar/navbar.jsx]: There are several other instances in the code that use the same class components and have the same clickable error. If you think it's appropriate, we can address their fixes in this PR. WDYT?

File: client/src/assets/shoessss.ico
Comment (dvjsharma) [inline client/src/assets/shoessss.ico:1]: Consider using better naming. Also, if this file is not used anywhere please remove it.

**Review by dvjsharma — CHANGES_REQUESTED** [review]
Changes needed, PTAL

**Comment by Code-eat-sleep-include** [comment]
@dvjsharma Okay, I'll do the necessary changes!

**Comment by Code-eat-sleep-include** [comment]
@dvjsharma I've made the requested changes! Changes the Fevicon with the existing logo. Removed the new logo from the navbar. Rest the signup fix and hover effect in navbar is as it is.

**Comment by dvjsharma** [comment]
Thanks @Code-eat-sleep-include. But I cannot see any new pushes/amends. Maybe you forgot to push the changes.

File: client/src/components/Navbar/navbar.jsx (hunk 1 above)
Comment (Code-eat-sleep-include) [reply client/src/components/Navbar/navbar.jsx]: > There are several other instances in the code that use the same class components and have the same clickable error. If you think it's appropriate, we can address their fixes in this PR. WDYT?

Yes I guess!

Comment (dvjsharma) [reply client/src/components/Navbar/navbar.jsx]: Sounds great. Ping us when you are done with the changes.

CC @ShashankShekhar07

**Comment by Code-eat-sleep-include** [comment]
> Thanks @Code-eat-sleep-include. But I cannot see any new pushes/amends. Maybe you forgot to push the changes.

Yeah sorry, I made a mistake in doing that. I'll do it again.

**Comment by Code-eat-sleep-include** [comment]
@dvjsharma Done!

File: client/src/components/Navbar/navbar.jsx (hunk 1 above)
Comment (Code-eat-sleep-include) [reply client/src/components/Navbar/navbar.jsx]: @dvjsharma It's done!

**Comment by ShashankShekhar07** [comment]
LGTM

**Pull request merged**